test/index.js

```diff
@@ -486,7 +486,7 @@ describe('Underdog', () => {
                 srv.stop(done);
             });
 
-            request.on('push', () => next(new Error('Should not make it here')));
```

Comment:
Returning an error wont do anything!  You'd have to at least throw it.  I think that `next` was possibly supposed to be `done`.

Reply:
ay, gotcha. thanks for pointing that out, slopslopslop. fixing now!